src/sentry/receivers/rules.py

```diff
@@ -31,7 +37,29 @@ def create_default_rules(project: Project, default_rules=True, RuleModel=Rule, *
         return
 
     rule_data = DEFAULT_RULE_DATA
-    RuleModel.objects.create(project=project, label=DEFAULT_RULE_LABEL, data=rule_data)
+    rule = RuleModel.objects.create(project=project, label=DEFAULT_RULE_LABEL, data=rule_data)
+
+    try:
+        user: RpcUser = project.organization.get_default_owner()
+    except IndexError:
+        logger.warning(
+            "Cannot record default rule created for organization (%s) due to missing owners",
+            project.organization_id,
+        )
+        return
```

Comment:
This will create a different behaviour than until now - previously we did not need an owner for an alert to be created, now we do - is this on purpose?

Reply:
When a new project is created, an issue alert is automatically created by default. However, we didn’t fire the `alert_rule_created` event (on purpose), so the quick start task wasn’t marked as completed. Now, we want to fire the event, and the function requires the user object.